validator/utils.py

```diff
@@ -13,13 +13,13 @@ def get_latest_version():
 
 
 REGEX_ASSETS_V2 = {
-    "assets_re": re.compile(r'.*INSERT +INTO +assets\( *identifier *, *type *, *name *, *symbol *, *started *, *swapped_for *, *coingecko *, *cryptocompare *, *details_reference *\) +VALUES\((.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?)\).*?'),
-    "ethereum_tokens_re": re.compile(r'.*INSERT +INTO +ethereum_tokens\( *address *, *decimals *, *protocol *\) +VALUES\((.*?),(.*?),(.*?)\).*'),
-    "common_asset_details_re": re.compile(r'.*INSERT +INTO +common_asset_details\( *asset_id *, *forked *\) +VALUES\((.*?),(.*?)\).*')
+    "assets_re": re.compile(r'.*INSERT +INTO +assets\( *identifier *, *type *, *name *, *symbol *, *started *, *swapped_for *, *coingecko *, *cryptocompare *, *details_reference *\) *VALUES\((.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?),(.*?)\).*?'),
+    "ethereum_tokens_re": re.compile(r'.*INSERT +INTO +ethereum_tokens\( *address *, *decimals *, *protocol *\) *VALUES\((.*?),(.*?),(.*?)\).*'),
+    "common_asset_details_re": re.compile(r'.*INSERT +INTO +common_asset_details\( *asset_id *, *forked *\) *VALUES\((.*?),(.*?)\).*')
 }
 
 REGEX_ASSETS_V3 = {
-    "assets_re": re.compile(r'.*INSERT +INTO +assets\( *identifier *, *name *, *type *\) +VALUES\(([^\)]*?),([^\)]*?),([^\)]*?)\).*?'),
-    "ethereum_tokens_re": re.compile(r'.*INSERT +INTO +evm_tokens\( *identifier *, *token_kind *, *chain *, *address *, *decimals *, *protocol *\) +VALUES\(([^\)]*?),([^\)]*?),([^\)]*?),([^\)]*?),([^\)]*?),([^\)]*?)\).*'),
-    "common_asset_details_re": re.compile(r'.*INSERT +INTO +common_asset_details\( *identifier *, *symbol *, *coingecko *, *cryptocompare *, *forked *, *started *, *swapped_for *\) +VALUES\((.*?),(.*?),(.*?),(.*?),(.*?),([^\)]*?),([^\)]*?)\).*')
-}
\ No newline at end of file
+    "assets_re": re.compile(r'.*INSERT +INTO +assets\( *identifier *, *name *, *type *\) *VALUES\(([^,]*?),([^,]*?),([^,]*?)\).*?'),
```

Comment:
Group is everything but comma now since before it was failing for strings that had parentheses in them. Like what this PR adds for a name of a token: `'Diversified Staked ETH Index (dsETH)'`

Reply:
Have you tested if the parser in the repo handles this?

Reply:
Which parser? There is a CI test that failed in this repo. This is why I adjusted this.

Reply:
This regular expressions that we have in the main repo https://github.com/rotki/rotki/blob/655631850fed261c59037c23dd12a0dcf8111fb6/rotkehlchen/globaldb/updates.py#L123

Reply:
Added it here: https://github.com/rotki/rotki/pull/7549